ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -123,13 +110,23 @@ open class DrawerController: UIViewController {
     }
 
     /// Set `backgroundColor` to customize background color of the drawer
-    @objc open var backgroundColor: UIColor = Colors.Drawer.background {
+    @objc open lazy var backgroundColor: UIColor = drawerBackgroundColor(fluentTheme: view.fluentTheme) {
         didSet {
             useCustomBackgroundColor = true
             view.backgroundColor = backgroundColor
```

Comment:
Hi Lamine, could you fix this issue in this same PR?

a check for `isViewLoaded` should be added before accessing the `view`, or it will cause the `loadView()` and `viewDidLoad()` to be called before other properties are set and some undesired behaviour could happen, like the presentationStyle be ignored.

```
if isViewLoaded {
    view.backgroundColor = backgroundColor
}
```

I filed an issue here: #1338 

I have a workaround, which is to set backgroundColor last, but it would be good to have this fixed for future versions.

Thanks!

Reply:
As discussed offline, I'll be making the fix in a different PR targeting the main branch to get it out in the next fluent release. Thank you for filing the issue!